guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-smart-proxy-server.adoc

```diff
@@ -37,15 +37,15 @@ If the command is successful, it returns two `{certs-generate}` commands, one of
 ----
 Validation succeeded.
 
-To use them inside a NEW {SmartProxyServer}, run this command:
-  {certs-generate} --foreman-proxy-fqdn "{SmartProxyServer}" \
+To use them inside a NEW ${smart-proxy-capitalized}, run this command:
```

Comment:
For context: it comes from https://github.com/theforeman/foreman-installer/blob/ee24dcce8b709d1badf4137f25ebd1ab8be7c9a4/bin/katello-certs-check#L276

The value is set here: https://github.com/theforeman/foreman-installer/blob/ee24dcce8b709d1badf4137f25ebd1ab8be7c9a4/bin/katello-certs-check#L22

And branding is loaded if present: https://github.com/theforeman/foreman-installer/blob/ee24dcce8b709d1badf4137f25ebd1ab8be7c9a4/bin/katello-certs-check#L11-L15

Reply:
Thanks; i've also added those links to my commit message.